packages/webpack/lib/configs/build.js

```diff
@@ -133,9 +135,21 @@ module.exports = function getConfig(config, name) {
           },
         }),
       ],
+      chunkIds: 'natural',
+
+      // TODO: re-consider setting this to false
+      //
+      // Currently certain modules are missing in the production bundle,
+      // like e.g. "@babel/runtime/helpers/esm/extend", which leads to
+      // __webpack_require__(null) being invoked which again leads to an error.
+      //
+      // This setting prevents this, but likely has its drawbacks, too, regarding
+      // bundle size.
+      concatenateModules: false,
     },
     plugins: [
-      new (isProduction ? HashedModuleIdsPlugin : NamedModulesPlugin)(),
+      new (isProduction ? HashedModuleIdsPlugin : NamedModuleIdsPlugin)(),
+      new DefinePlugin({ 'process.env': JSON.stringify({}) }),
```

Comment:
Why do we need this?

Reply:
`hops-bootstrap/lib/utils` introduces `process.env` as a fallback when `global._env` is undefined. In the browser build this won't ever happen, but in the worker build there's no global `_env` variable. And since webpack 5 does not polyfill Node.js-related thingies anymore, we have to provide this ourselves. I can add a comment.